hathor_tests/poa/test_poa_simulation.py

```diff
@@ -158,7 +158,7 @@ def test_two_producers(self) -> None:
         assert self.simulator.run(200, trigger=trigger)
         assert manager1.tx_storage.get_block_count() == 12
         assert manager2.tx_storage.get_block_count() == 12
-        assert manager1.tx_storage.get_best_block_tips() == manager2.tx_storage.get_best_block_tips()
+        assert manager1.tx_storage.get_best_block_hash() == manager2.tx_storage.get_best_block_hash()
```

Comment:
Should we keep the `get_best_block_tips()` that should return the best chain together with all candidate chains?